debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsStreamProducer.java

```diff
@@ -418,17 +429,21 @@ protected void generateDeleteRecord(TableId tableId, Object[] oldRowData, Blocki
         Map<String, ?> offset = sourceInfo.offset();
         String topicName = topicSelector().topicNameFor(tableId);
         Envelope envelope = tableSchema.getEnvelopeSchema();
+        Struct source = sourceInfo.source();
```

Comment:
From the javadoc of `source()`: 
> This method should always be called after {@link #update(Long, Instant, Long, TableId, Long)}.

Is it safe to call this method without first calling `update()`?